orttraining/orttraining/test/python/orttraining_test_ort_module_pytorch_ddp.py

```diff
@@ -0,0 +1,160 @@
+# This test script is a modified version of Pytorch's tutorial.
```

Comment:
Would you please change the name of the file to follow template ```orttraining_test_ortmodule_test_nam```? Jus remove the ```_``` here.

Reply:
Np.